SpockOpaleye/src/OpaleyeDef.hs

```diff
@@ -0,0 +1,173 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module OpaleyeDef where
+
+import Data.List.NonEmpty
+import Data.Profunctor.Product
+import qualified Data.Profunctor.Product.Default as D
+import Data.Text
+import Data.Text.Encoding
+import Database.PostgreSQL.Simple.FromField
+import Opaleye
+
+import DataTypes
+
+type TenantTableW = (Maybe (Column PGInt4), Column PGText, Column PGText, Column PGText, Column PGText, Column PGText, Column PGText, Maybe (Column (Nullable PGInt4)), Column PGText)
```

Comment:
shouldn't we be using records for this? can you quickly tell what's the difference between the first `PGText` column and the second?